test/SequentialContainerTest/Deque.test.ts

```diff
@@ -115,6 +117,15 @@ describe('Deque test', () => {
     expect(new Deque(a).length).to.equal(a.size());
 
     const q = new Deque<number>([], (1 << 10));
+    const v = new Vector<number>();
+    for (let i = 0; i < (1 << 15); ++i) {
+      const random = Math.random();
```

Comment:
No, this is what i did on purpose. If I use a random seed to generate the same tests data, some problem will be hidden forever.

It is very difficult to list all the cases, even if the test coverage reaches 100%, it does not mean that we have listed all the cases, I need as much data as possible to test, otherwise the risk is huge.

You can find we do tests many times for each PR and each has a different test data.